utils/formats.js

```diff
@@ -1,40 +1,36 @@
-const helpers = require('./helpers');
+const assetsURL = 'https://zesty-storage-prod.s3.amazonaws.com/images/zesty';
```

Comment:
We should probably put this endpoint behind AWS CloudFront/Cloudflare to enable caching

Reply:
Can definitely look into this yeah